modules/spo-log-enricher-app-trace.adoc

```diff
@@ -31,14 +31,22 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: log-pod
+  securityContext:
+   runAsNonRoot: true
+   seccompProfile:
+     type: RuntimeDefault
 spec:
   securityContext:
```

Comment:
@s-urbaniak @stlaz This has a securityContext set under spec (not spec.containers) - let me know if we need to adjust somehow

Reply:
see https://github.com/openshift/openshift-docs/pull/62472/files#r1266818096